sdk/core/core-client/src/deserializationPolicy.ts

```diff
@@ -12,10 +12,12 @@ import {
   OperationRequest,
   OperationResponseMap,
   FullOperationResponse,
-  OperationSpec
+  OperationSpec,
+  SerializerOptions
 } from "./interfaces";
 import { MapperTypeNames } from "./serializer";
 import { isStreamOperation } from "./interfaceHelpers";
+import { XmlOptions, XML_CHARKEY } from "@azure/core-xml";
```

Comment:
hm this is tricky, because one of the design requirements is that `core-client` doesn't have a runtime dependency on `core-xml` but this isn't a type-only import because of `XML_CHARKEY`

Reply:
Yes that's right! Maybe duplicating would work.

Reply:
I chose to duplicate since this is small amount of code, and core-xml only have two dependencies: tslib and xml2js. It doesn't feel worth it to add a common dependency or shared source.